### PR TITLE
Functional test suit amendment to address RM and LM 

### PR DESCRIPTION
**Pull request by msvinaykumar**
## Description

Functional test suite amendment to address RM and LM .
List_recommendations is called using flag rm=true from remote monitoring test cases.

Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] New feature
- [ ] Docs update
- [ ] Breaking change (What changes might users need to make in their application due to this PR?)
- [ ] Requires DB changes

## How has this been tested?

Please describe the tests that were run to verify your changes and steps to reproduce. Please specify any test configuration required. 

- [ ] New Test X
- [x] Functional testsuite

**Test Configuration**
* Kubernetes clusters tested on: 

## Checklist :dart:

- [x] Followed coding guidelines
- [x] Comments added
- [x] Dependent changes merged
- [x] Documentation updated
- [ ] Tests added or updated

## Additional information

Include any additional information such as links, test results, screenshots here

### CODE REVIEW

File: tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py [hunk 1]
```diff
@@ -122,7 +124,7 @@ def main(argv):
     experiment_name = None
     latest = "false"
     interval_end_time = None
-    response = list_recommendations(experiment_name, latest, interval_end_time)
+    response = list_recommendations(experiment_name, latest, interval_end_time, rm=True)
```
Comment (chandrams) [inline tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py:127]: @msvinaykumar - This change should be made in the remote monitoring fault tolerant test.

Comment (msvinaykumar) [reply tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py:127]: done

Comment (chandrams) [reply tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py:127]: Where is this change done? I still see rm=True in local monitoring fault tolerant test.

**Comment by chandrams** [comment]
@msvinaykumar - PR check has failed, can you take a look.

File: tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py (hunk 1 above)
Comment (msvinaykumar) [reply tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py:127]: This change is necessary for Fault Tolerance. Are you suggesting removing it from this PR and including it in the Fault Tolerance PR instead? Or is this change not required at all?

**Comment by msvinaykumar** [comment]
> @msvinaykumar - PR check has failed, can you take a look.

done

File: tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py (hunk 1 above)
Comment (chandrams) [reply tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py]: We have fault tolerant test in both local monitoring and remote monitoring.  This change I think is applicable only for remote monitoring fault tolerant test.

**Comment by dinogun** [comment]
@msvinaykumar Please rebase this PR

**Comment by dinogun** [comment]
@msvinaykumar Please rebase

**Comment by msvinaykumar** [comment]
> @msvinaykumar Please rebase this PR

done

File: tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py (hunk 1 above)
Comment (msvinaykumar) [reply tests/scripts/local_monitoring_tests/fault_tolerant_test/kruize_pod_restart_test.py]: Got it, thank you. I removed this change, but the file was still checked in due to a formatting issue.

**Review by chandrams — APPROVED** [review]
LGTM

**Pull request merged**